hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/LogAdapter.java

```diff
@@ -17,61 +17,40 @@
  */
 package org.apache.hadoop.util;
 
-import org.apache.commons.logging.Log;
 import org.slf4j.Logger;
 
 class LogAdapter {
```

Comment:
this class is obsolete. tag as deprecated and see if all uses can be removed.

Reply:
Done, removed all usages. Maybe we can remove it in another sub-task.